.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: CI & CD
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Is there a standard way of setting things up so it only publishes on tagged commits? Do we need separate github action workflows for testing and publishing? I haven't been using cibuildwheel so I'm not sure what the common practices are.

Reply:
Yes, it's possible to publish only on tagged commits – will update it :+1: And we do not need a separate workflow for testing and publishing. This all happens inside of cibuildwheel and the tests run against every wheel (e.g. Python 3.11 and 3.12 on Linux):

![image](https://github.com/lda-project/lda/assets/16133277/38f94e67-4b58-4bff-ae97-9f0634d8f3c5)

The tests all passed in the [dry run of my fork](https://github.com/severinsimmler/lda/actions/runs/6866391327):

![image](https://github.com/lda-project/lda/assets/16133277/bcb0a079-0c0b-421e-a1ee-62b7efd0eaff)